src/screens/CreateToken.js

```diff
@@ -187,6 +192,25 @@ class CreateToken extends React.Component {
     this.setState({amount: e.target.value});
   }
 
+  /**
+   * Method called when user clicked to see the token deposit RFC
+   *
+   * @param {Object} e Event for the click
+   */
+  goToRFC = (e) => {
+    e.preventDefault();
+    const url = "https://gitlab.com/HathorNetwork/rfcs/blob/master/text/0011-token-deposit.md";
```

Comment:
I think we should move it to the constants file.

Reply:
Done